finality-aleph/src/new_network/session.rs

```diff
@@ -0,0 +1,115 @@
+use crate::{
+    crypto::{AuthorityPen, AuthorityVerifier},
+    new_network::{ComponentNetwork, SendError, Sender as SenderTrait, SessionCommand},
```

Comment:
Isn't naming convention a little misleading? "Sender" means 3 different things depending on the context in this PR....

Reply:
Definitely, since reading this code requires one to check what is named what in which file, which is not good :)

Reply:
As above.